src/components/container/Container.js

```diff
@@ -21,11 +21,11 @@ export const Container = ({
     return (
       <SafeAreaView
       style={[
-        containerStyle, style.circleProgressIndicator,
+        containerStyle, style.circleActivityIndicator,
         {backgroundColor: background, paddingHorizontal: horizontalPadding},
       ]}
       {...rest}>
-        <ProgressIndicator.Circle size={size.xl} indeterminate={true} />
+        <ActivityIndicator size='large' color={color.primary} />
```

Comment:
Should it be size={size.lg} instead?

Reply:
It's not using the size file. The ActivityIndicator has a built in size prop that takes 'small' or 'large'. It is very easy to switch back to the external library version that gives some more size flexibility if required.